pymc3/tests/test_gp.py

```diff
@@ -1177,3 +1177,17 @@ def test_plot_gp_dist_warn_nan(self):
             )
         plt.close()
         pass
+
+
+class TestCircular:
+    def test_1d(self):
+        X = np.linspace(0, 1, 10)[:, None]
+        with pm.Model():
+            cov = pm.gp.cov.Circular(1, 1, ls=1)
```

Comment:
I think you say in the doc strings that `tau`'s minimum value is 4, but here it takes value 1 -- am I mistaken?